requirements.txt

```diff
@@ -2,8 +2,9 @@
 # pip install -r requirements.txt
 lxml
 requests
-rdflib
-rdflib-jsonld
+rdflib>=6.0.1; python_version>="3.7"
+rdflib<6.0.0; python_version<"3.7"
+rdflib-jsonld; python_version<"3.7"
```

Comment:
I think we need to specify an extra constraint here and in setup.py, as according to https://pypi.org/project/rdflib-jsonld/

> If you are forced to keep using Python <= 3.6, you will need to keep using release <= 0.5.0 of this plugin with RDFlib 5.0.0.

I hope that would also fix the tests.